inference/models/utils.py

```diff
@@ -59,6 +64,63 @@
     ("object-detection", "yolov10m"): YOLOv10ObjectDetection,
     ("object-detection", "yolov10l"): YOLOv10ObjectDetection,
     ("object-detection", "yolov10x"): YOLOv10ObjectDetection,
+    ("object-detection", "yolov11"): YOLOv11ObjectDetection,
```

Comment:
Can you make `yolo11***` (without the v) also be aliased to these?

Reply:
these aren't aliases, these are the model types coming from the platform. when adding aliases I will make sure to include both forms